board/flasher.h

```diff
@@ -137,11 +138,10 @@ void soft_flasher_start(void) {
   // enable USB
   usb_init();
 
-  // enable SPI
-  if (current_board->has_spi) {
```

Comment:
Should probably keep that check, technically you can build with `ENABLE_SPI` set without having a spi-compatible board